## <span style="color:darkblue"> Problem Set 7: Meet Mr. $t$


**Problem set policies:** Please provide concise, clear answers for each question. Note that only writing the result of a calculation (e.g., "SD = 3.3") without explanation is not sufficient. For problems involving `R`, include the code in your solution, along with any plots.
		
*You are encouraged to discuss problems with other students (and, of course, with the course head and the TAs), but you must write your final answer in your own words. Solutions prepared "in committee" are not acceptable. If you do collaborate with classmates on a problem, please list your collaborators on your solution.*

### Problem 1:

An independent random sample is selected from an a population. Test the hypothesis with a $t$-test, by finding the p-value for the given 
$H_0: \mu=\mu_0$, $n=$ number of samples, and $T=\frac{\overline x - \mu}{SE}$. Assume $\alpha =0.05$. 

(a) $H_A :\mu > \mu_0$, $n=11$, $T =1.91$


In [1]:
pt(-1.91,10)

[1] 0.04260244

Answer: Since $p=4.26\%<5\%$ we would reject $H_0$.

(b) $H_A :\mu < \mu_0$, $n=17$, $T =−3.45$

In [3]:
pt(-3.45,16)

[1] 0.001646786

Answer: Since $p=0.16\%<5\%$, we would reject $H_0$.

 (c) $H_A :\mu\neq\mu_0$, $n=7$, $T =0.83$

In [5]:
2*pt(-0.83,6)

[1] 0.4383084

Answer: Since $p=43.8\%>5\%$, we cannot reject $H_0$.

(d) $H_A :\mu > \mu_0$, $n=28$, $T =2.13$

In [6]:
pt(-2.13,27)

[1] 0.02121769

Answer: Since $p=2.12\%<5\%$ we would reject $H_0$.

### Problem 2

A $95\%$ confidence interval for a population with  mean $\mu$, is given as $(18.985, 21.015)$. This confidence interval is based on a simple random sample of $36$ observations. Calculate the sample mean and standard deviation. Assume that all conditions necessary for inference are satisfied. Use the $t$-distribution in any calculations.

In [9]:
#The mean is the middle of the confidence interval
m=(21.015-18.985)/2
#We can get the standard error by undoing the margin or error qt(.975,35)*SE
SE=(21.015-m)/qt(.975,35)
#then the standard deviation is sqrt(36)*SE
s=6*SE

"sample mean" 
m
"std dev" 
s

[1] "sample mean"

[1] 1.015

[1] "std dev"

[1] 59.11016

### Problem 3

In each of the following scenarios, determine if the data are paired. 

(a) Compare pre- (beginning of semester) and post-test (end of semester) scores of students. 



Answer: Paired, because you would pair scores of the same students.

(b) Assess gender-related salary gap by comparing salaries of randomly sampled men and women. 

Answer: Independent, because the salaries of the men, *in theory*, should not effect the salaries of the women.

(c) Compare artery thicknesses at the beginning of a study and after 2 years of taking Vitamin E for the same group of patients.

Answer: Paired, because we would pair the arteries of the same patient

(d) Assess effectiveness of a diet regimen by comparing the before and after weights of subjects.

Answer: Paired, because we would pair the weights of the same subject.

### Problem 5

Is there strong evidence of global warming? Let’s consider a small scale example, comparing how temperatures have changed in the US from 1968 to 2008. The daily high temperature reading on January 1 was collected in 1968 and 2008 for 51 randomly selected locations in the continental US. Then the difference between the two readings (temperature in 2008 - temperature in 1968) was calculated for each of the 51 different locations. The average of these 51 values was 1.1 degrees with a standard deviation of 4.9 degrees.  Do these data provide strong evidence of temperature warming in the continental US? Conduct a hypothesis test; interpret your conclusions in context. (Use $t$-test)

In [1]:
#start with the one-sided version.
x = 1.1
SE = 4.9/sqrt(51)
T = (x-0)/SE
"one-sided p value is"
pt(-T,50)

#now the two-sided
x = 1.1
SE = 4.9/sqrt(51)
T = (x-0)/SE
"two-sided p value is"
2*pt(-T,50)

[1] "one-sided p value is"

[1] 0.05759731

[1] "two-sided p value is"

[1] 0.1151946

Answer: The null hypothesis is $H_0: \delta = 0$, and the alternative hypothesis is either $H_A: \delta>0$ or $H_A: \delta\neq 0$. We will test both in turn.

This is paired data because it is in the same places.

The onesided $p$-value is $5.8\%$ which is just bigger than $5\%$, which means we would not reject the null hypothesis.

The two sided hypothesis has $p$-value of $11.5\%$, which is bigger than $5\%$, and so is not strong evidence.